src/main/java/com/sonyericsson/jenkins/plugins/bfa/db/MongoDBKnowledgeBase.java

```diff
@@ -274,10 +271,12 @@ private void initCache() {
         keys.put("modifications", 1);
         keys.put("lastOccurred", 1);
         BasicDBObject orderBy = new BasicDBObject("name", 1);
-        DBCursor<FailureCause> dbCauses =  getJacksonCollection().find(NOT_REMOVED_QUERY, keys);
-        dbCauses = dbCauses.sort(orderBy);
-        while (dbCauses.hasNext()) {
-            list.add(dbCauses.next());
+        final FindIterable<FailureCause> dbCauses = getJacksonCollection().find(NOT_REMOVED_QUERY);
```

Comment:
why is this one not using the new filter? NOT_REMOVED_QUERY_FILTER like on line 254
https://github.com/jenkinsci/build-failure-analyzer-plugin/pull/136/files#diff-2a90eeca96b68dde8fc30cb2662b2a6567e8cbebe1ac970b3875161b2b70a005R254

Reply:
Looks like this one and the one below are just leftovers, I'll change to the new filter and remove the old one.